user_sync/connector/directory_ldap.py

```diff
@@ -86,6 +86,22 @@ def __init__(self, caller_options):
         username = builder.require_string_value('username')
         builder.require_string_value('base_dn')
         options = builder.get_options()
+
+        if options['two_steps_lookup']:
```

Comment:
This doesn't seem to be requiring `group_member_attribute_name` if the two-step option is enabled.  For example, sync runs if I explicitly set `two_steps_lookup` to an empty dictionary object.

```yaml
two_steps_lookup: {}
```

Checking that `options['two_steps_lookup'] is not None` should validate that an empty object isn't getting set.